packages/next-swc/crates/next-core/src/next_import_map.rs

```diff
@@ -583,10 +583,9 @@ async fn insert_next_server_special_aliases(
 
             rsc_aliases(import_map, project_path, ty, runtime, next_config).await?;
         }
-        ServerContextType::Middleware => {
```

Comment:
These turbopack changes are required but not enough to support the new tests, going to follow up in a new PR to handle the transform for middleware and instrumentation